plugins/jobtype/src/azkaban/jobtype/HadoopJavaJob.java

```diff
@@ -121,8 +121,27 @@ protected String getJVMArguments() {
     classPath.add(HadoopConfigurationInjector.getPath(getJobProps(),
         getWorkingDirectory()));
 
-    List<String> typeClassPath =
-        getSysProps().getStringList("jobtype.classpath", null, ",");
+    // merging classpaths from plugin.properties
+    mergeClassPaths(classPath,
+      getJobProps().getStringList("jobtype.classpath", null, ","));
```

Comment:
No there is no constant for jobtype.classpath. I thinking changing "jobtype.classpath" with constant for all the plugins will be a much bigger refracting change which we should take separately.